### DISCUSSION TITLE
iOS - Errors out when trying to view bookmarked/saved notifications

### DISCUSSION THREAD

**Discussion by joshdholtz** [community/community#3353] [Mobile] [answered]
1) I love this app so much! Thank you, team ❤️ 

2) It looks like there are some issues happening with viewing bookmarked/saved notifications. It initially shows some things and then it shows it failed. “Try again” doesn’t seem to fix anything 🙃 Below is a video of this happening 👇 

Also... thanks for adding video upload support! 🙌 

cc: @rnystrom 

https://user-images.githubusercontent.com/401294/107532985-152c1e00-6b84-11eb-98c0-29e83eb0cfbd.MP4

**Comment by rnystrom** [comment, marked as answer]
Yup I can reproduce this, thankfully it's only affecting TestFlight builds. We will fix!

**Reply by joshdholtz** [reply]
Woot! Thanks for looking into so quickly ❤️ 

Also, was this the proper channel to report this? 😇

**Reply by rnystrom** [reply]
Ya this works well 😄

**Reply by rnystrom** [reply]
This is now fixed! Was a bit more complicated than we imagined. h/t @eliperkins for owning it
